Update AdvancedOperators.md (#587)
AcademyIFCE/Swift-Book: Sources/TSPL/TSPL.docc/LanguageGuide/AdvancedOperators.md

  ```
}

### Precedência para operadores infixos personalizados

Cada operador infixo personalizado pertence a um grupo de precedência.
Um grupo de precedência especifica a precedência de um operador em relação
a outros operadores infixos, bem como a sua associatividade.
Consulte <doc:AdvancedOperators#Precedence-and-Associativity> para obter uma explicação sobre
como essas características afetam a interação de um operador infixo
com outros operadores infixos.

> Um operador infixo personalizado que não é colocado explicitamente em um grupo de precedência recebe um grupo de precedência padrão com uma precedência imediatamente maior
> do que a precedência do operador condicional ternário.

Reviewer: oq vc acha da sugestão abaixo?

```suggestion
Se um operador infixo personalizado não for explicitamente colocado em um grupo de precedência, lhe é atribuído um grupo de precedência padrão com uma precedência imediatamente maior
do que a precedência do operador condicional ternário.
```

O exemplo a seguir define um novo operador infixo personalizado chamado `+-`,

> que pertence ao grupo de precedência `AdditionPrecedence`:

Reviewer: ```suggestion
que pertence ao grupo de precedência `_AdditionPrecedence_`:
```

let firstVector = Vector2D(x: 1.0, y: 2.0)
let secondVector = Vector2D(x: 3.0, y: 4.0)
let plusMinusVector = firstVector +- secondVector

> // 'plusMinusVector' é uma instância Vector2D com valores de (4.0, -2.0)

Reviewer: ```suggestion
// 'plusMinusVector' é uma instância de Vector2D com valores de (4.0, -2.0)
```

  ```
}

Este operador soma os valores `x` de dois vetores,
e subtrai o valor `y` do segundo vetor do primeiro.
Porque é essencialmente um operador "aditivo",
foi dado o mesmo grupo de precedência
dos operadores infixos aditivos como `+` e `-`.

> Para obter informações sobre os operadores fornecidos pela biblioteca padrão Swift,

Reviewer: ```suggestion
Para obter informações sobre os operadores fornecidos pela biblioteca padrão do Swift,
```

incluindo uma lista completa dos grupos de precedência do operador e configurações de associatividade,
consulte  [Operator Declarations](https://developer.apple.com/documentation/swift/operator_declarations).

> Para obter mais informações sobre grupos de precedência e para ver a sintaxe de
> definição dos seus próprios operadores e grupos de precedência,

Reviewer: ```suggestion
Para obter mais informações sobre grupos de precedência e ver a sintaxe para
definir seus próprios operadores e grupos de precedência,
```
